docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.6
+
+COPY restic /usr/bin
```

Comment:
@quantonganh https://github.com/restic/restic/blob/master/Dockerfile install Golang and bunch of dependencies for build and testing. that are totally unrelated to run the binary/daemon. My docker image is super small compared to the other, it's just alpine + restic binary.